pkgs/os-specific/linux/kernel/linux-rt-5.10.nix

```diff
@@ -6,7 +6,7 @@
 , ... } @ args:
 
 let
-  version = "5.10.186-rt91"; # updated by ./update-rt.sh
+  version = "5.10.180-rt89"; # updated by ./update-rt.sh
```

Comment:
To the eye this looks like a downgrade?

Reply:
The patch got taken down upstream with no explanation over a week ago, and no one responded when I pinged people about it, so I think we should keep it at this point.